Doc/library/string.rst

```diff
@@ -716,12 +721,14 @@ Replacing ``%x`` and ``%o`` and converting the value to different bases::
    >>> "int: {0:d};  hex: {0:#x};  oct: {0:#o};  bin: {0:#b}".format(42)
    'int: 42;  hex: 0x2a;  oct: 0o52;  bin: 0b101010'
 
-Using the comma or the underscore as a thousands separator::
+Using the comma or the underscore as a digit separator::
```

Comment:
Agreed that this is not the correct terminology but would `digit group` separator be more correct (just suggesting as non-native).

Reply:
Should I replace `digit separator` with `digit group separator` only here or in other places as well?

Reply:
Sure, I added "group" everywhere. However, for type `n` I removed "characters" as "digit group separator characters" felt a bit off (4 nouns in a row).

> I don't know if you're a native
No, I'm not native.